apps/builder/app/builder/features/topbar/entri.tsx

```diff
@@ -0,0 +1,157 @@
+import useScript from "react-script-hook";
```

Comment:
lets move most of the logic for entri in a single package and keep here only whats specific about topbar